src/firefox/index.js

```diff
@@ -248,9 +329,18 @@ export async function useProfile(
   {
     app,
     configureThisProfile = configureProfile,
+    isFirefoxDefaultProfile = isDefaultProfile,
     customPrefs = {},
   }: UseProfileParams = {},
 ): Promise<FirefoxProfile> {
+  const isForbiddenProfile = await isFirefoxDefaultProfile(profilePath);
```

Comment:
@saintsebastian @kumar303 I'm open to implement such option if we decide to go for it, but I vote against it :-)